addons/mail/static/src/chatter/web/thread_service_patch.js

```diff
@@ -0,0 +1,19 @@
+/* @odoo-module */
+
+import { ThreadService } from "@mail/core/common/thread_service";
+import "@mail/chatter/web_portal/thread_service_patch";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(ThreadService.prototype, {
+    async fetchData(
+        thread,
+        requestList = ["activities", "followers", "attachments", "messages", "suggestedRecipients"]
```

Comment:
I still feel overriding default params is really confusing (and can't work if we need multiple independent overrides).
It would probably be better to simply have defaults stored as a static field and have patch push on that list.
Or have a separate method to return default values, and override it.